terraform/templates/defaults/ecs_taskdef.tpl

```diff
@@ -42,16 +42,28 @@
             "value": "${sample_app_listen_address}"
         },
         {
-        "name": "JAEGER_RECEIVER_ENDPOINT",
-        "value": "127.0.0.1:${http_port}"
+            "name": "JAEGER_RECEIVER_ENDPOINT",
+            "value": "127.0.0.1:${http_port}"
         },
         {
-        "name": "ZIPKIN_RECEIVER_ENDPOINT",
-        "value": "127.0.0.1:${http_port}"
+            "name": "ZIPKIN_RECEIVER_ENDPOINT",
+            "value": "127.0.0.1:${http_port}"
         },
         {
-        "name": "OTEL_METRICS_EXPORTER",
-        "value": "otlp"
+            "name": "OTEL_METRICS_EXPORTER",
+            "value": "none"
```

Comment:
Why have you disabled the metrics exporter?

Reply:
Tests passed, but got this error message in cloudwatch:
`[otel.javaagent 2023-10-31 16:59:10:056 +0000] [OkHttp http://127.0.0.1:4317/...] ERROR io.opentelemetry.exporter.internal.grpc.GrpcExporter - Failed to export metrics. Server responded with UNIMPLEMENTED. This usually means that your collector is not configured with an otlp receiver in the "pipelines" section of the configuration. If export is not desired and you are using OpenTelemetry autoconfiguration or the javaagent, disable export by setting OTEL_METRICS_EXPORTER=none. Full error message: unknown service opentelemetry.proto.collector.metrics.v1.MetricsService`
So, what do you think? Keep it or get rid of it?

Reply:
Just leave it. You are adjusting the default template for all ecs test cases. Other test cases may require that, there is a large blast radius in that template.